tests/wamr-test-suites/test_wamr.sh

```diff
@@ -1044,7 +1078,7 @@ if [[ $TEST_CASE_ARR ]];then
     trigger || (echo "TEST FAILED"; exit 1)
 else
     # test all suite, ignore polybench and libsodium because of long time cost
-    TEST_CASE_ARR=("spec")
+    TEST_CASE_ARR=("spec" "malformed" "wasi" "sightglass" "standalone")
```

Comment:
Similarly, maybe it's best to keep the original set, some of the added ones are unavailable